.github/workflows/backport-pr-fixes-validation.yaml

```diff
@@ -22,5 +22,12 @@ jobs:
             const regex = new RegExp(pattern);
             
             if (!regex.test(body)) {
-              core.setFailed("PR body does not contain a valid 'Fixes' reference.");
+              const error = "PR body does not contain a valid 'Fixes' reference.";
+              core.setFailed(error);
+              await github.rest.issues.createComment({
+                issue_number: context.issue.number,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                body: `:warning: ${error}`
+              });
```

Comment:
Not all pull requests fix a bug.

Reply:
But this is what you asked to verify for releases in https://github.com/scylladb/scylla-pkg/issues/3539

Reply:
Fixes only (or Fix or Fixed)

Reply:
but we might have fixes which do not fully address a bug, so they are not qualified to have a `Fixes` tag, but we do want to get them backported. in that case, may we use `Refs` ?

Reply:
@tchaikov We should use any of GitHub keyword only ( see https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)

Reply:
@yaronkaikov the keyword is used to close an issue, and github support it. why can't we use a different tag, if we don't want to close an issue with the tag?

Reply:
Well, if you don't want to close an issue, then maybe we can use `Refs`, but automation will not take it under consideration, so for example, we will not update the backport label once it's done (will send a patch to Scylla next week)

Reply:
yes, i explained the rationales in this thread on under what circumstances we can use "Refs". if by "automation", what you meant is the backport, i think we *might* want to take the PRs with "Refs" tag into consideration, as they could prepare for the fix which have the "Fixes" tag, in other words, without them, the final fix might not apply.